operators/pkg/instanceset-landing/security.go

```diff
@@ -0,0 +1,72 @@
+package instanceset_landing
+
+import (
+	"crypto/hmac"
+
+	// nolint:gosec // used in polito api
+	"crypto/sha1"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+func keyedShaFromString(str string) string {
+	h := hmac.New(sha256.New, []byte(Options.InternalKey))
+
+	_, err := h.Write([]byte("SEC-" + str + "-CL"))
+	if err != nil {
+		klog.Error("SHA256 generation error [%v]", err)
+	}
+
+	return fmt.Sprintf("%x", h.Sum(nil))
+}
+
+func sha1FromString(in string) string {
+	h := sha1.New() // nolint:gosec // used in polito api
+	_, err := h.Write([]byte(in))
+	if err != nil {
+		klog.Error("SHA256 generation error [%v]", err)
+	}
+
+	return fmt.Sprintf("%x", h.Sum(nil))
+}
+
+func generateIstanceIdentifier(matr, cCode string) string {
```

Comment:
Please, be consistent with the parameters names. Would also change the method to checkIdentifierValidity, or something like that.